### PR TITLE
members list no longer adds MEPS

### PR DESCRIPTION
**Pull request by abarker66**
The update members list no longer includes UK MEPs as this list is now empty.

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/info/api/MemberService.java [hunk 1]
```diff
@@ -49,7 +49,6 @@ void updateWebMemberLists() {
         updateMember(listConsumerService.createCommonsFromUKParliamentAPI());
         updateMember(listConsumerService.createLordsFromUKParliamentAPI());
         updateMember(listConsumerService.createFromIrishAssemblyAPI());
-        updateMember(listConsumerService.createFromEuropeanParliamentAPI());
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/info/api/MemberService.java:52]: Might as well remove anything to do with the European Parliament API. We won't require this I shouldn't have thought, plus if we need to revert it will be in Git history.

**Review by LiamMacP — CHANGES_REQUESTED** [review]

**Comment by zuzak** [comment]
A [better Git commit message](https://chris.beams.io/posts/git-commit/) would be good.
Try to instruct the computer what to do (`Remove MEPs from updateMembers`) so it matches better with auto-generated Git commit messages. We also tend to start the first line of the Git commit with the relevant Jira ticket reference.

Ideally a commit message would go into detail as to _why_ a commit was made -- we know it's removing MEP code because we can look at the commit itself. A developer in six months might not know why we removed MEP code: in this case, it's because the MEP code can't handle zero UK MEPs existing.

**Comment by abarker66** [comment]
I find it curious that there is so much concern about my git comments when the class in question has public methods with no access modifiers (cardinal Java sin), no exception handlers for methods handling object mapping!!!, and of course the method itself that cannot handle an empty xml list , neither do any of the remaining services - surely the unit tests would have picked that up being a valid edge condition.
Anyway, rant over, all trace of the EU parliament has been removed form the Member Service and associated classes.

**Comment by abarker66** [comment]
Pull request updated. These changes address the original issue of UK MEP list being empty and raising exceptions. It does not fic the fact that the remaining members services are vulnerable to empty list and missing or unavailable endpoints.

**Review by LiamMacP — APPROVED** [review]
Changes made, looks right.
